gammapy/maps/hpx.py

```diff
@@ -1687,7 +1687,13 @@ def get_idx(self, idx=None, local=False, flat=False):
 
     def get_coord(self, idx=None, flat=False):
         pix = self.get_idx(idx=idx, flat=flat)
-        return self.pix_to_coord(pix)
+        coords = self.pix_to_coord(pix)
+        cdict = OrderedDict(lon=coords[0], lat=coords[1])
```

Comment:
This doesn't enforce an ordering for lon/lat.  I'd suggest changing this to:
```cdict = OrderedDict([('lon',coords[0]),('lat',coords[1])])```

Reply:
Actually, the wrong order for lon / lat causes test fails in some builds, e.g. here:
https://travis-ci.org/gammapy/gammapy/jobs/373824783#L2215

I'll fix those OrderedDict calls now.